Block/Checkout/Success.php

```diff
@@ -140,13 +141,12 @@ public function getBoletoData()
      */
     public function getBankTransferData()
     {
-        $result = [];
-        if (!empty($this->getOrder()->getPayment()) &&
-            !empty($this->getOrder()->getPayment()->getAdditionalInformation('bankTransfer.owner'))
-        ) {
-            $result = $this->getOrder()->getPayment()->getAdditionalInformation();
-        }
-
+        $action = $this->getOrder()->getPayment()->getAdditionalInformation('action');
+        $result['beneficiary'] = !empty($action['beneficiary']) ? $action['beneficiary'] : '';
+        $result['iban'] = !empty($action['iban']) ? $action['iban'] : '';
+        $result['bankName'] = !empty($action['bankName']) ? $action['bankName'] : '';
+        $result['bic'] = !empty($action['bic']) ? $action['bic'] : '';
+        $result['reference'] = !empty($action['reference']) ? $action['reference'] : '';
```

Comment:
Since the component will render the bank data this is not used anymore. Should we remove it completely?

Reply:
Removed since the component will take care of it